<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1097</width>
    <height>780</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>142</width>
     <height>188</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_1">
      <property name="text">
       <string>Waga[kg]</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="Line_Edit_Waga">
      <property name="text">
       <string>80</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Wzrost[m]</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="Line_Edit_Wzrost">
      <property name="text">
       <string>1.7</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Push_Button_Oblicz">
      <property name="text">
       <string>Oblicz</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>BMI :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="Line_Edit_Bmi"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>701</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QWidget" name="widget_3" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="Combo_Box_Type_Sex"/>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Sex</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="Combo_Box_Type_Age"/>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Age</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="Combo_Box_Type_Goal"/>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Goal</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="Combo_Box_Type_Activity_Level"/>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Activity level</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_4" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Calories</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Line_Edit_Calories"/>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Fat</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Line_Edit_Fat"/>
       </item>
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Carbs</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Line_Edit_Carbs"/>
       </item>
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Protein</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Line_Edit_Protein"/>
       </item>
       <item>
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Basal metabolic rate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Line_Edit_Bmr"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Push_Button_Calculate_Macronutrients">
      <property name="text">
       <string>Calculate macronutrients</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_6" native="true">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>0</y>
     <width>381</width>
     <height>321</height>
    </rect>
   </property>
   <widget class="QWidget" name="widget_9" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>351</width>
      <height>83</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Choose:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_5" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Muscle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="Combo_Box_Type_Muscles"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_7" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Exercise</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="Combo_Box_Type_Exercises"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_8" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Difficulty</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="Combo_Box_Type_Difficulties"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="Push_Button_Find_Exercise">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Find Exercise</string>
    </property>
   </widget>
   <widget class="QLabel" name="Line_Edit_Exercises">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="widget_10" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>291</width>
      <height>181</height>
     </rect>
    </property>
    <widget class="QPushButton" name="Push_Button_Calculate_Orm">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>150</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Calculate One Rep Max</string>
     </property>
    </widget>
    <widget class="QLabel" name="Qlabel_Orm">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>99</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="Line_Edit_Weight_Lifted">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>100</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Line_Edit_Reps">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Weight Lifted[kg]</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>161</x>
       <y>40</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Reps[count)</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>370</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Account</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Line_Edit_Name_User">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Jacek</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Line_Edit_Surname_User">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>BBBBB</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Line_Edit_Weight_User">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>82</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Line_Edit_Height_User">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>550</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1.62</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Line_Edit_Sex_User">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>600</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>male</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Line_Edit_Age_User">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>650</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>45</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>400</y>
     <width>42</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>440</y>
     <width>42</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Surname</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>480</y>
     <width>42</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Weight</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>530</y>
     <width>42</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Height</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>580</y>
     <width>42</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Sex</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>630</y>
     <width>42</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Age</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Push_Button_Create_User">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>690</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Create User</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Line_Edit_Exercise_User">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>390</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Leg Press</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Line_Edit_Reps_User">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>450</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Line_Edit_Weight_User_2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>520</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Line_Edit_Series_User">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>580</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>560</y>
     <width>42</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Series</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_26">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>490</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Weight lifted</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_27">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>430</y>
     <width>42</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Reps</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_28">
   <property name="geometry">
    <rect>
     <x>681</x>
     <y>370</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Name of Exercise</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Push_Button_Create_Exercise">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>610</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Exercise</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
